### PR TITLE
Update oic.if.ll schema for OCF 1.0 Content Format

### PR DESCRIPTION
**Pull request by mjkoster**
Changed the oic.if.ll format to array type, removing the "links" property tag in the response payload

### CODE REVIEW

File: schemas/oic.collection.linkslist-schema.json [hunk 1]
```diff
@@ -7,10 +7,5 @@
 		       "$ref": "oic.collection-schema.json#/definitions/oic.collection.alllinks"
     }
   },
-  "type": "object",
-  "properties": {
-    "links":  {
-      "$ref": "#/definitions/oic.collection.alllinks"
-    }
-  }
+  "$ref": "#/definitions/oic.collection.alllinks"
```
Comment (marktrayer) [inline schemas/oic.collection.linkslist-schema.json]: I don't think you can just leave a $ref floating like that.  Wrap it in an allOf block; i.e.
"allOf": [
  {"$ref": "#/definitions/oic.collection.alllinks"}
]

Comment (mjkoster) [reply schemas/oic.collection.linkslist-schema.json]: Yes, thanks!

**Comment by jinchoe** [comment]
Michael 

I assume the changes are 
- modify slinks (i.e., oic.collection.linkslist-schema.json)  from JSON object to an array of Links 
- change accordinlgy the example in oic.wk.col.raml. 

Do I understand right? All look fine, thanks. 

best regards

JinHyeock

File: schemas/oic.collection.linkslist-schema.json (hunk 1 above)
Comment (RamTJ) [reply schemas/oic.collection.linkslist-schema.json]: Is deletion of "type": "object" in line 10 intentional?

Comment (marktrayer) [reply schemas/oic.collection.linkslist-schema.json]: Yes.  This now an array of links and so this schema just $refs to alllinks in the collections schema that is just that; an array of links.  The schema no longer represents a JSON object.

**Pull request merged**